### ISSUE TITLE
Project Management: Hosting site for development and testing

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#301]
Following our call on Tuesday and issue #297 we discussed the option of moving the hosting of FarmOS to platform.sh or similar so we could build a better environment for testing before pushing changes. With the increasing complexity of the Rothamsted version of FarmOS this seems increasingly useful/ relevant.

We also discussed moving to a two week sprint rather than what is now (approximately) a one week sprint. This would give me more time to test and document the issues related to every release.

**The cost estimates Paul gave for hosting in [Platform.sh](https://platform.sh/):** 

$50 per instance per month (circa £42.50) although from Autumn he thinks this price might drop to $25 for companies hosting only a small number of instances (circa £22.60)

Duplicated and updated issue for:
#164

**Comment by aislinnpearson** [comment]
We agreed to move the Rothamsted instance over to platform.sh. @paul121 will do the initial set up, and then I'll arrange for the regular monthly billing. We agreed this will mostly likely be done by the end of the year as it takes a while to set up but will move it between milestones until it is done to keep track of it

**Comment by aislinnpearson** [comment]
Hey @paul121,

I just want to confirm this issue _only_ relates to the monthly billing right? As far as I am concerned, platform.sh is now functioning as our .dev site.

But I wanted to double check with you before actioning

**Comment by paul121** [comment]
Yeah I believe that is correct. Right now I'm paying for this hosting and
including on my invoices. Its ~$25 but it depends on how many concurrent
demos are running, more than 3 and it costs a little more.

We got close to switching over the "ownership" of the demo site... I think
there is a Rothamsted Organization, but we can't move the project over
until billing is configured. It would be nice to get this done, but no rush
on my end.

**Comment by aislinnpearson** [comment]
@paul121 this is done my end - sorry it was such a prolonged delay.

**Comment by paul121** [comment]
Okay, I've done some testing to confirm I can still deploy to the environments after changing ownership, and can confirm all is working! Thanks!

Just thinking now, perhaps another thing we could do is move this Github repo over to the Rothamsted organization. This is the repo that configures the platform.sh environments (separate branches/PRs can be created for each environment). It also has some notes on how to restore a backup from Farmier to the platformsh test environment. https://github.com/paul121/rothamsted-site

**Comment by aislinnpearson** [comment]
This was moved to the Rothamsted Eco-informatics github repo: https://github.com/Rothamsted-Ecoinformatics/rothamsted-site
